src/Summoner/CLI.hs

```diff
@@ -122,18 +120,14 @@ versionP = infoOption summonerVersion
 summonerVersion :: String
 summonerVersion = toString $ unlines [sVersion, sHash, sDate, sDirty]
```

Comment:
It seems better to write `toText` only once instead of adding `toText` before every variable:

```haskell
summonerVersion = toString $ unlines $ map toText [sVersion, ...]
```